frontend/src/app/modules/work_packages/routing/work-packages-routes.ts

```diff
@@ -133,6 +134,15 @@ export const WORK_PACKAGES_ROUTES:Ng2StateDeclaration[] = [
       menuItem: menuItemClass
     }
   },
+  {
+    name: 'work-packages.show.additionalTabs',
+    url: '/additionalTabs/:additionalTabIdentifier',
```

Comment:
idea for later 💡 

@oliverguenther what if we remove the `additionalTabs` namespace here - we may override following routes (and be careful to place this last), but we may also get better support for `updateFocusAnchorLabel`

Reply:
we settled on moving this into the `tabs` namespace and eventually (in the future) move other tabs there as well